### ISSUE TITLE
How to change SSID via telnet?

### ISSUE THREAD

**Issue by Vendo232** [HclX/WyzeHacks#144]
Would anyone know how to change SSID and password on wyze v3 via telnet?

**Comment by endertable** [comment]
Yes, there are 2 files in the /configs directory.  One is named .wifissid and the other is named .wifipasswd and each holds the relevant info.
As an example type the following to put a new SID/password:

cd /configs
echo -n “newsid” > .wifissid
echo -n “newpassword” > .wifipasswd

Then reboot.
